angular/core/components/sidebar/sidebar.coffee

```diff
@@ -47,8 +47,10 @@ angular.module('loomioApp').directive 'sidebar', ($rootScope, $mdMedia, $mdSiden
       InboxService.unreadCount()
 
     $scope.showContactUs = ->
-      # TODO: use loomio_org plugin to determine official site or not
-      AppConfig.baseUrl == 'https://www.loomio.org/'
+      IntercomService.available()
```

Comment:
I think this changes behaviour a little bit; we have the contact form as a fallback if people have disabled intercom for whatever reason (as a 3rd party script).

Anyway, we should always display contact us, and switch what happens when you click it based on whether Intercom is available or not.

Reply:
Yea.. it's because I noticed we don't support contact by email right now. I'd prefer that too.